source/index.html.md

```diff
@@ -5294,6 +5298,410 @@ Payment Requests where you're the creditor and collecting funds from the debtor.
 |---|---|---|---|
 |200|[OK](https://tools.ietf.org/html/rfc7231#section-6.3.1)|OK|[ListOutgoingPaymentRequestsResponse](#schemalistoutgoingpaymentrequestsresponse)|
 
+## List Payment Requests: Collections
+
+<a id="opIdListOutgoingPaymentRequestCollections"></a>
+
+> Code samples
+
+```shell
+curl --request GET \
+  --url https://api.sandbox.split.cash/payment_requests/collections \
+  --header 'accept: application/json' \
+  --header 'authorization: Bearer {access-token}'
+```
+
+```ruby
+require 'uri'
+require 'net/http'
+
+url = URI("https://api.sandbox.split.cash/payment_requests/collections")
+
+http = Net::HTTP.new(url.host, url.port)
+http.use_ssl = true
+http.verify_mode = OpenSSL::SSL::VERIFY_NONE
+
+request = Net::HTTP::Get.new(url)
+request["accept"] = 'application/json'
+request["authorization"] = 'Bearer {access-token}'
+
+response = http.request(request)
+puts response.read_body
+```
+
+```javascript--node
```

Comment:
I haven't seen this language syntax before. Is it specific to the API doc publishing tool which we use?

AFAIK Github markdown uses this:
https://github.com/github/linguist/blob/master/lib/linguist/languages.yml

... which doesn't support `javascript--node`:

```javascript--node
var http = require("https");
```

... but works with `javascript`:

```javascript
var http = require("https");
```
... and `node`:

```node
var http = require("https");
```